extensions/matts_luxury_pack/templates/default/css/main.css

```diff
@@ -3418,9 +3418,29 @@ div.colCopy.ie div {
 	border-radius: 5px;
 }
 .subnav li a {
-    border-bottom: 3px solid transparent;
+/*    border-bottom: 3px solid transparent;
+    border-top: 3px solid transparent;*/
+    border: 3px solid transparent;
 }
 .subnav li a:hover {
     border-bottom: 3px solid #D54E21;
 }
+/**/
+#Container {
+	display: none;
+}
+#pageLoading {
+	display: block;
+	position: absolute;
+	top: 0;
+	left: 0;
+	z-index: 100;
+	width: 100vw;
+	height: 100vh;
+	background-color: rgba(192, 192, 192, 0.5);
+	background-image: url("http://i.stack.imgur.com/MnyxU.gif");
```

Comment:
The reference to ````http://i.stack.imgur.com/MnyxU.gif```` at line 3441 in  ````extensions/matts_luxury_pack/templates/default/css/main.css```` can be localised by providing the image in the extension itself.
![mnyxu](https://cloud.githubusercontent.com/assets/2995044/18191417/c5856424-70e7-11e6-92e7-ba2ee1502646.gif)
or better still use the smaller sized **png** version below:
![mnyxu](https://cloud.githubusercontent.com/assets/2995044/18191473/5bdbc2a6-70e8-11e6-8bc8-874c97a6b0a2.png)
Use [gif2apng](http://gif2apng.sourceforge.net/) to convert all animated and static **gif** to **png** files.